storage/etcdv2/service.go

```diff
@@ -74,11 +74,9 @@ type Service struct {
 	prefix string
 }
 
-func (s *Service) Create(ctx context.Context, key, value string) error {
-	_, err := s.keyClient.Create(ctx, s.key(key), value)
-	if IsEtcdKeyAlreadyExists(err) {
-		return nil
-	} else if err != nil {
+func (s *Service) Set(ctx context.Context, key, value string) error {
+	_, err := s.keyClient.Set(ctx, s.key(key), value, &client.SetOptions{})
```

Comment:
`Set` instead of `Create` is used to align with etcd v3 and in-memory semantics which is "create or update", not "create or fail when exists".